thrift/src/main/java/com/linecorp/armeria/internal/thrift/ThriftServiceMetadata.java

```diff
@@ -56,8 +60,21 @@
      * interfaces.
      */
     public ThriftServiceMetadata(Object implementation) {
-        requireNonNull(implementation, "implementation");
```

Comment:
Revert this?

Reply:
I think I removed because` ImmutableList.of ` checks null. But I think it will be more user-friendly if we have explicit check as well. :)

Reply:
I cannot use any statement before `this`. But ImmutableList.of does a null check.

Reply:
Yes, 👍 I know `ImmutableList.of()` checks null, we prefer to check ourselves on user-facing public methods even though it could be delegated.
https://github.com/line/armeria/blob/master/CONTRIBUTING.md#check-null

Reply:
Added requireNonNull

Reply:
Got it @ikhoon, that makes total sense. It is always best to catch null as early as possible. Thanks for the tip.  Thanks @trustin for making this change.